net/HttpRequest.hpp

```diff
@@ -1376,7 +1378,7 @@ class Session final : public ProtocolHandlerInterface
     }
 
     /// Returns the socket FD, for logging/informational purposes.
-    int getFD() const { return _fd; }
+    int fd() const { return _fd; }
```

Comment:
I have to admit this might be just another misunderstanding here .. my fault.
First I added the accessor for `_fd` with name `getFD()`. Then we had a chat here regarding naming and whether accessors are good/useful. Then I probably wrongly assumed (?) to use simpler naming to imply no processing in such accessor (removing the verb get). 
Yeah, at this stage this added to the confusion and review - bad decision on my behalf. Sorry.

Reply:
reverted